Dockerfile

```diff
@@ -1,21 +1,22 @@
-FROM lsstdesc/stack-sims:w_2019_42-sims_w_2019_42-v2
+FROM lsstdesc/stack-sims:w_2019_42-sims_w_2020_24
```

Comment:
This should be 
FROM lsstdesc/stack-sims:w_2020_24-sims_w_2020_24

Reply:
I'm leaving you a reply